app/templates/application.hbs

```diff
@@ -1,46 +1,37 @@
-<div id="nav">
   <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top" >
-    <a class="navbar-brand" href="#">CoolWriter</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+    {{#link-to 'index'}}<span class="navbar-brand">CoolWriter</span> {{/link-to}}
```

Comment:
Translation is missing. Consider handlebars linter that should prevent such issues.

Reply:
CoolWriter is name of the app so i think it should remain static

Reply:
Every bare string should end up in translations. Imagine the case where the name of your app is offensive in some language - then you have to adjust it. It is also a good practice to hold everything there so you get use to it - also template linters can fail your CI because of that.